README.md

```diff
@@ -41,6 +41,16 @@ elastic:
 
 ```
 
+To change the default url prefix (`/api`) add a new key-value at the top level:
```

Comment:
nitpick: I think its not prefix. rather endpoint or sth

Reply:
The value I'm setting in Flask is "url_prefix", so while I think that would probably be better in a general sense, I would rather stick with the Flask name.
See here: https://github.com/tomtom-international/ebr-board/pull/22/files#diff-6d442ccbe058f7bbb9751d9a2d95cab2L44